### PR TITLE
DOC: Add Jupyter Lab IndexError as known issue upstream

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to document a known upstream issue that @javerbukh reported at https://github.com/spacetelescope/jdaviz/pull/1373#issuecomment-1150001663 . Also see jupyterlab/jupyterlab#11934 .

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [x] Is a milestone set?
- [x] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/known_bugs.rst [hunk 1]
```diff
@@ -61,6 +62,23 @@ in bottleneck trying to build numpy from source and crash, stalling the
...
+.. _known_issues_app:
+
+Application-wide
+----------------
+
+Lab fails to start with IndexError
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In some environment, you might not able to start Jdaviz in Jupyter Lab
+and see this error occasionally::
```
Comment (rosteen) [inline docs/known_bugs.rst:74]:
```suggestion
In some environments, you occasionally might not able to start Jdaviz in 
Jupyter Lab due to this error ::
```

**Review by rosteen — APPROVED** [review]
One suggestion on phrasing to (I think) make it a little clearer, otherwise looks good. Thanks for adding this!

File: docs/known_bugs.rst (hunk 1 above)
Comment (pllim) [reply docs/known_bugs.rst]: Yes, this is better. Thanks!

**Comment by pllim** [comment]
Thanks for the review!

**Pull request merged**